.github/workflows/code-checks.yml

```diff
@@ -10,13 +10,16 @@ concurrency:
   group: ${{ github.workflow }}-${{ github.ref }}-${{ github.head_ref }}
   cancel-in-progress: ${{ ! (github.ref == 'refs/heads/master' || startsWith(github.ref, 'refs/heads/release/')) }}
 
+defaults:
+  run:
+    shell: bash
+
 jobs:
+
   mypy:
     runs-on: ubuntu-20.04
-    # TODO: package parametrization
```

Comment:
I think this TODO should stay. There might be mypy differences in the mirror vs the standalone